components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -1464,11 +1499,20 @@ public void removeTokenIdCookie(HttpServletRequest req, HttpServletResponse resp
                     diagnosticLog.info("samlssoTokenId Cookie is removed");
                     samlSsoTokenIdCookie.setHttpOnly(true);
                     samlSsoTokenIdCookie.setSecure(true);
-                    samlSsoTokenIdCookie.setPath("/");
+
+                    boolean isTenantQualifiedCookie = false;
+                    if (IdentityTenantUtil.isTenantedSessionsEnabled() &&
+                            cookie.getValue().endsWith(SAMLSSOConstants.TENANT_QUALIFIED_TOKEN_ID_COOKIE_SUFFIX)) {
+                        samlSsoTokenIdCookie.setPath(FrameworkConstants.TENANT_CONTEXT_PREFIX + loginTenantDomain + "/");
```

Comment:
Check character number  - max 120

Reply:
Can't we use stringUtils concat method

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/348